### DISCUSSION TITLE
Expand/collapse indicator on left side of menu?

### DISCUSSION THREAD

**Discussion by robertmaybin** [thegeeklab/hugo-geekdoc#600] [Ideas]
By default, the expand/collapse menu indicator is on the right side of collapsible menu items, denoted with `<`.

If there are number of collapsible menus listed one after the other in the menu structure, the net effect is a vertical column of `<` symbols between the menu area and the currently displayed post.  This continually pulls the eye away from the content to the menu, and is visually distracting/confusing.

<img width="176" alt="image" src="https://user-images.githubusercontent.com/22859626/227284277-f01d7113-71ba-4d61-93af-7677f8ba9d33.png">

Is there an existing configuration option to move the expand/collapse indicators to the left side of the menu?

If not, this would be a welcome enhancement to the theme!

**Comment by xoxys** [comment]
Right now, there is no way built-in to the theme to change it. Moving the foldable icon to the left is not easily possible, as this conflicts with custom icons that can be used in bundle menus.
